packages/compass-aggregations/src/components/pipeline-toolbar/pipeline-header/index.tsx

```diff
@@ -169,21 +148,18 @@ export const PipelineHeader: React.FunctionComponent<PipelineHeaderProps> = ({
   );
 };
 
-
 export default connect(
   (state: RootState) => {
     return {
       isOpenPipelineVisible: !state.editViewName && !state.isAtlasDeployed,
       isSavedPipelineVisible: state.savedPipeline.isListVisible,
+      // TODO: this component should not be the one connected to the saved
+      // pipelines state as it's not the one using it
```

Comment:
I'm leaving some clean-up things out to avoid making the diff bigger, but will revisit right after